packages/react-storage/src/components/StorageBrowser/createStorageBrowser/__tests__/createStorageBrowser.test-d.tsx

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable react/jsx-no-useless-fragment */
+import { expectAssignable, expectNotAssignable } from 'jest-tsd';
+import createStorageBrowser from '../createStorageBrowser';
+import { defaultActionConfigs } from '../../actions';
+import React from 'react';
```

Comment:
3rd party imports should be above local imports. Module to be asserted against should bt the final omport